### PR TITLE
[ch21204] - Helm chart v0.2.1

### PR DESCRIPTION
**Pull request by rodrigo-findhotel**
- changed the `apiversion` from ingress service
- changed the `helm chart` version from `v0.2.0` to `v0.2.1`
- readme file updated
- added `kubectl` command to remove the pod test created by the `helm test`.

[ch21204](https://app.clubhouse.io/findhotel/story/21204/check-the-memory-metrics-on-the-hpa-file-in-the-helm-chart)

Pipeline:
https://g.codefresh.io/build/5f476d8aeb2749e06ed65950?step=deploy&tab=output&logs=text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,8 @@ stable release (CD pipeline). During development the Docker image is tagged with
 and `branch name` (CI pipeline).
 
 The Helm Chart version uses SemVer for stable releases, but during development the version
-is always `0.0.1:{SHORT_COMMIT_HASH}`.
\ No newline at end of file
+is always `0.0.1:{SHORT_COMMIT_HASH}`.
+
+## How to use the development version
```
Comment (victorpoluceno) [inline README.md:15]: 🥇

File: charts/README.md [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+How to Install
+--------------
+
+This is an example of `how to install`.
+
+```
+helm install CHART_NAME cf-review-env \
+--install \
+--reset-values \
+--repo cm://h.cfcr.io/findhotel/default/ \
```
Comment (victorpoluceno) [inline charts/README.md:17]: Don't forget that this project is public. Better to replace this repo and the ecr repo and anything that is private from here.

File: deploy/deploy [hunk 3]
```diff
@@ -63,6 +63,12 @@ helm_test_status=$(kubectl get pods --selector="app.kubernetes.io/managed-by=Hel
 if [[ "$helm_test_status" == "Failed" ]]; then
   echo "ERROR: The branch has not passed because the Helm Test failed, please check your application pod.";
   exit 1;
+else
+  # It is necessary to remove the test pod from the helm test bacause HPA metrics may fail when we have a pod with completed status.
```
Comment (victorpoluceno) [inline deploy/deploy:67]: Well done.

**Review by victorpoluceno — COMMENTED** [review]
Left a small comment.

File: charts/README.md (hunk 2 above)
Comment (rodrigo-findhotel) [reply charts/README.md]: done!

**Review by victorpoluceno — APPROVED** [review]

**Pull request merged**